web/packages/teleport/src/Assist/Chat/ChatItem/Action/Action.tsx

```diff
@@ -173,7 +175,10 @@ export function Action(props: ActionProps) {
       {!editing && (
         <Buttons>
           <EditButton onClick={() => setEditing(true)}>
-            <EditIcon size={18} />
+            <EditIcon
+              size={18}
+              fill={theme.name === 'light' ? 'black' : 'white'}
```

Comment:
I'm aware this isn't an ideal check as we may introduce more themes in the future, however this is a common approach at the moment and I expect any future change to be a sweeping update